### ISSUE TITLE
Angular 2 Router

### ISSUE THREAD

**Issue by kentoj** [AngularJSUtah/meetups#42]
The Angular 2 component router shown at NG-Conf was very powerful and seemed simple to use, once you became familiar with it.

However, since NG-Conf the router has undergone some development and now the router is on v3. It sounds like things have somewhat stabilized so now seems to be a good time to get in deep with the Angular 2 router.

This link is the link to the pull request for the developer guide to the new router:
⁃ https://github.com/angular/angular.io/pull/1636/files

This link talks about routing in angular:
⁃ http://victorsavkin.com/post/145672529346/angular-router

This link is from Thoughtram, posted yesterday, talking about the v3 router:
- http://blog.thoughtram.io/angular/2016/06/14/routing-in-angular-2-revisited.html

How many would be interested in a talk on this topic?

**Comment by mwinche** [comment]
👍

**Comment by jasonwaters** [comment]
hey @kentoj, would you like to speak about the Angular 2 Router at an upcoming meetup?  How about September 13th?

**Comment by jspenc72** [comment]
@kentoj I would love to hear you talk about the new router.

**Comment by jspenc72** [comment]
The new forms module would be a good one as well.

**Comment by jspenc72** [comment]
@jasonwaters we've been using the new Router for about 1 month now and could probably be ready to present in another 2 weeks if anyone is interested but if @kentoj is willing to present that would be a great talk well worth attending.

**Comment by jasonwaters** [comment]
I added another issue for angular 2 forms as well, great suggestion!

https://github.com/AngularJSUtah/meetups/issues/47

**Comment by kentoj** [comment]
Yes, I would like to speak on that. I’ll talk with Jesse and see whether doing a collaborative presentation would work, or something like that.
September 13th will work for me.

**Comment by jasonwaters** [comment]
Hey @kentoj, I'm curious if you'd be interested in presenting on this topic (or forms) at the next meetup on November 8th?  We'd love to have you.

**Comment by kentoj** [comment]
I could speak about my current knowledge of the router, which is not much at the moment. I am looking forward to reading Victor Savkin's book on the subject after which I would be much more prepared to speak on the subject and could provide a greater audience experience. Also, November 8 will not work for me.
Would next month work? I could shoot for a 20-30 minute deep dive presentation on the new router.
What do you think?
